src/agent/src/config.rs

```diff
@@ -343,6 +352,12 @@ impl AgentConfig {
                 config.guest_components_procs,
                 get_guest_components_procs_value
             );
+            parse_cmdline_param!(
+                param,
+                IMAGE_REGISTRY_AUTH_OPTION,
+                config.image_registry_auth,
+                get_string_value
+            );
```

Comment:
@stevenhorsman, excuse my ignorance here, but is this a path to a file on the host side? on the guest side?

Reply:
The guest side, there is no mounting, or copyfile involved. It's actually a uri, so could be `file:///run/confidential-containers/auth.json`, or `kbs://default/credentials/test`

Reply:
That does remind me that we have a config doc now, so I'll find that and add the info there

Reply:
To maybe answer the question behind the question - the primary use case of the file uri is likely to be peer pods where we have the ability to copy the image pull secret and write it into the guest via cloud-init.

Reply:
> That does remind me that we have a config doc now, so I'll find that and add the info there

Added - let me know if that would have helped answer the question, or if more clarity is needed. Thanks!

Reply:
> To maybe answer the question behind the question - the primary use case of the file uri is likely to be peer pods where we have the ability to copy the image pull secret and write it into the guest via cloud-init.

Yeah. that's exactly what I was trying to figure out. :-)

Not to digress much, but what are the other mechanisms that could be used for this when used on baremetal / on-prem?
I assume that asking folks to rebuild the rootfs is a no-go.

Reply:
The bare-metal case is to use the `kbs` uri to get this info from the remote kbs, which uses the same parameter